.eslintrc.json

```diff
@@ -39,6 +39,7 @@
         "import/no-named-as-default-member": "off",
         "import/no-named-as-default": "off",
         "no-console": ["error", { "allow": ["warn", "error"] }],
-        "quotes": ["error", "single"]
+        "quotes": ["error", "single"],
+        "react/react-in-jsx-scope": "off"
```

Comment:
can we add this as `"plugin:react/jsx-runtime"` extension as recommended on rule doc (see last paragraph) https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md